spec_parser/utils.py

```diff
@@ -330,7 +333,18 @@ def __init__(
         self.logger = logging.getLogger(self.__class__.__name__)
         self._extract_metadata(metadata)
         self._extract_properties(props)
-# TODO: handle ext_props in some way -- for now, silently ignored
+        self.format_pattern = format_pattern
+        self.ext_props = ext_props
+        if ext_props:
+            self.logger.warning("External property restrictions aren't yet handled properly, they are added to the "
+                                "description of the class.")
+            self.description += "\nExternal property restriction:"
```

Comment:
This probably should render the `ext_props`?

Reply:
added